plugins/quota/api/req.go

```diff
@@ -0,0 +1,13 @@
+package quota
+
+// LimitReq represents REST API request to Limit Usage/objects of a directory
+type LimitReq struct {
```

Comment:
You can name this as `SetLimitReq` and the other one as `RemoveLimitReq`

Reply:
Agreed!